virustotal_yara/help.md

```diff
@@ -1,13 +1,14 @@
 # Description
 
-The VirusTotal Yara plugin allows the user to analyze files with the Yara Python libarary.
-
-[VirusTotal Yara](http://virustotal.github.io/yara/) is a pattern matching swiss knife for malware researchers.
```

Comment:
We should add the original link

Reply:
moved down the paragraph, but never removed.